client/lib/numalib/detect_linux.go

```diff
@@ -89,7 +89,10 @@ func (*Sysfs) discoverCosts(st *Topology, readerFunc pathReaderFn) {
 		}
 
 		for i, c := range strings.Fields(s) {
-			cost, _ := strconv.Atoi(c)
+			cost, _ := strconv.ParseUint(c, 10, 8)
+			if err != nil {
+				return err
+			}
```

Comment:
the error is ignored in the parse but then we check it's not nil? this error has nothing to do with the parsing

Reply:
Oops, fixed that too.